product.html

```diff
@@ -44,50 +44,47 @@
             <h2>Trójmiejski Kalendarz Imprez</h2>
             <h3>Najlepsza wyszukiwarka - najlepsze Imprezy</h3>
 
-            <!--Odtąd dodane opis planowanych funkcji produktu-->
-            <!--modyfikacja: dodać podział na 4 okienka z wyraxnym tytułem i krotkim rozwinięciem.
-            Opis może być rozwijany-->
             <div class="product-container">
                 <hr>
                 <h4>Planowane funkcje produktu</h4>
```

Comment:
może zamiast hrów dać border top i bottom w css?

Reply:
Na kolejnym etapie dynamizacji strony wezmę to pod uwagę.